### DISCUSSION TITLE
Syntax highlighting

### DISCUSSION THREAD

**Discussion by mattgreenfield** [open-southeners/vscode-laravel-pint#25] [Ideas]
Would it be possible to highlight errors in the code (red underlines)?
Is it within the scope of this project?

**Comment by d8vjork** [comment]
I already thought about this but there's a huge problem with this...

The problem is PHP-CS-Fixer (the one Laravel Pint depends on) doesn't report with line numbers

Issues here:

https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/3601#issuecomment-419066854
https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/4968

**Reply by mattgreenfield** [reply]
ah, that's a shame! Thanks for your response

**Reply by d8vjork** [reply]
Yeah, no worries mate! Let's see if they change their minds after or either Laravel's team push them to do this
